jerry-core/debugger/debugger-ws.h

```diff
@@ -38,18 +38,12 @@ typedef struct
 /**
  * Byte data for evaluating expressions and receiving client source.
  */
-typedef struct
+typedef struct jerry_debugger_uint8_data_t
```

Comment:
This change seems to be unjustified. Current code base adds names to those structs only that are recursive. Here, it's not the case.